generator/gen_1.0.0.go

```diff
@@ -0,0 +1,170 @@
+package generator
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"sort"
+
+	"github.com/onflow/cadence/runtime/parser"
+	"github.com/onflow/flixkit-go"
+	"github.com/onflow/flow-cli/flowkit"
+	"github.com/onflow/flow-cli/flowkit/config"
+	"github.com/onflow/flow-cli/flowkit/gateway"
+	"github.com/onflow/flow-cli/flowkit/output"
+	"github.com/onflow/flow-go-sdk/crypto"
+	"github.com/spf13/afero"
+)
+
+type Generator1_0_0 struct {
+	deployedContracts []flixkit.Contracts
+	coreContracts     flixkit.Contracts
+	testnetClient     *flowkit.Flowkit
+	mainnetClient     *flowkit.Flowkit
+}
+
+// stubb to pass in parameters
+func NewGenerator(deployedContracts []flixkit.Contracts, coreContracts flixkit.Contracts, logger output.Logger) (*Generator1_0_0, error) {
+	loader := afero.Afero{Fs: afero.NewOsFs()}
+
+	gwt, err := gateway.NewGrpcGateway(config.TestnetNetwork)
+	if err != nil {
+		return nil, fmt.Errorf("could not create grpc gateway for testnet %w", err)
+	}
+
+	gwm, err := gateway.NewGrpcGateway(config.MainnetNetwork)
+	if err != nil {
+		return nil, fmt.Errorf("could not create grpc gateway for mainnet %w", err)
+	}
+
+	state, err := flowkit.Init(loader, crypto.ECDSA_P256, crypto.SHA3_256)
+	if err != nil {
+		return nil, fmt.Errorf("could not initialize flowkit state %w", err)
+	}
+	testnetClient := flowkit.NewFlowkit(state, config.TestnetNetwork, gwt, logger)
+	mainnetClient := flowkit.NewFlowkit(state, config.MainnetNetwork, gwm, logger)
+
+	if coreContracts == nil {
+		coreContracts = getDefaultCoreContracts()
+	}
+
+	return &Generator1_0_0{
+		deployedContracts: deployedContracts,
+		coreContracts:     coreContracts,
+		testnetClient:     testnetClient,
+		mainnetClient:     mainnetClient,
+	}, nil
+}
+
+func (g Generator1_0_0) Generate(code string) (*flixkit.FlowInteractionTemplate, error) {
+	template := &flixkit.FlowInteractionTemplate{}
+
+	// strip out imports in case invalid cadenece code is provided with imports that have placeholders like 0xFUNGIBLETOKENADDRESS
```

Comment:
If we are processing an interaction template that has imports in a given way I think this should lead to an error. There is a lot of code to handle that this is not the case and I do not think it is worth it.

Is this something that will be done for 1.1 aswell? I see all VIT here https://github.com/onflow/flow-interaction-template-service/blob/master/templates/flow-transfer-tokens.template.json use this old format.

Have we considered just transforming them to the new import syntax here instead and use the existing tools in flowkit?

Reply:
There is an importReplacer in flixkit that could easily be used here if we just set things up correct first.

Reply:
FCL loading templates, only supports placeholders (old syntax) in templates for 1.0.0. I'll support the new import syntax in 1.1.0. 

I wanted to support 1.0.0 as a stop gap until we get all the infra in for 1.1.0